### PR TITLE
Deprecate Private Access and Logging - use awcli instead

### LINKED ISSUE #1714: [AWS] Deprecate Private Access and Logging
We didn't add private access and logging control plane settings on EKS at the time we move from shell based kfctl to golang based kfctl. 

This is pretty straightforward setting on EKS side, we don't have to integrate into kfctl. The plan is to have these docs there but change the `awscli` way rather than configs in `cluster_feature.yaml`

### PR DESCRIPTION
**Pull request by Jeffwan**
Resolve #1714

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1715)

### CODE REVIEW

File: content/docs/aws/logging.md [hunk 1]
```diff
@@ -6,34 +6,42 @@ weight = 70
...
 * api
 * audit
 * authenticator
 * controllerManager
 * scheduler
 
-Open the [CloudWatch Console](https://console.aws.amazon.com/cloudwatch/home#logs:prefix=/aws/eks). Choose the cluster that you want to view logs for. The log group name format is `/aws/eks/${AWS_CLUSTER_NAME}/cluster`.
+Run command to enable logs
+```shell
+aws eks --region us-west-2 update-cluster-config --name <your_eks_cluster_name> \
```
Comment (sarahmaddox) [inline content/docs/aws/logging.md]: Is `<your_eks_cluster_name` the same as `${AWS_CLUSTER_NAME}` used in other commands, and if so would it be clearer to use the env variable here too?

Comment (Jeffwan) [reply content/docs/aws/logging.md]: Yes. We should reuse env variable here. I will make the change

**Comment by Jeffwan** [comment]
Address the feedback and please have another check

**Comment by sarahmaddox** [comment]
@Jeffwan Shall I add this PR and related issue to the **doc sprint** count? (I know that you and the team were working on code changes during the sprint week, and that you had to hold off on the doc changes until the code was merged.)

**Comment by Jeffwan** [comment]
for sure!

**Pull request merged**